components/schemas/scoped-variables/ScopedVariableAccess.yml

```diff
@@ -1,25 +1,35 @@
 title: ScopedVariableAccess
 type: object
-description: The type or way the scoped variable is accessed.
+description: The way the scoped variable is accessed.
 required:
   - env_variable
 properties:
   env_variable:
     type: boolean
-    description: A boolean, where true represents this scoped variable is accessed like a normal environment variable.
+    description: When set to true, this scoped variable is set as an environment variable inside the container.
   internal_api:
     nullable: true
+    description: Descibes an object with variables that the internal api will serve after runtime starts
```

Comment:
This needs updated to something like "If set, this scoped variable will be available over the internal API. Contains settings for accessing this variable over the internal API."

Reply:
Updated to that ^